setup.py

```diff
@@ -295,6 +301,9 @@ def add_unique(l: List[str], vals: Union[str, List[str]]) -> None:
             add_unique(setup_reqs, "pybind11")
         add_unique(install_reqs, "tensorflow")
         add_unique(test_reqs, ["keras", "tensorflow_datasets"])
+    if "paddle" in frameworks():
+        # add_unique(install_reqs, "paddle")
```

Comment:
I commented it out because pip always tried to reinstall paddle when installing TE. I found out later it was because the actual package name for paddle is `paddlepaddle-gpu`, not `paddle`. I will uncomment and modify accordingly.